### DISCUSSION TITLE
Question about `yarn link`

### DISCUSSION THREAD

**Discussion by wind4gis** [yarnpkg/berry#4239] [Q&A]
i use yarn berry to develop a library, however my project uses yarn 1.x, i can't upgrade those project's yarn version, how can i debug my library ? i often used `yarn link`, but yarn berry remove link, how can i debug a yarn berry library in yarn 1.x projects ?

can you support yarn link in yarn berry ? or a yarn link plugin ?

**Comment by pringshia** [comment]
I have a somewhat similar use case. The fact that `yarn link` modifies the package.json in the client project doesn't work perfectly for us. It would be nice to be able to do something similar to previous behavior where maybe `yarn link` adds a symlink in the `node_modules` folder for the client project that points back to the yarn berry workspace.
